src/components/Registration/index.jsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react'
+import { useLocation, Link } from 'wouter'
+import registrateService from '../../services/registrate'
+import Eye from '../Icons/Eye'
+import IMAGES from '../../images/images'
+
+export default function Registration () {
+    const [email, setEmail] = useState('')
+    const [fullname, setFullnameHandler] = useState('')
+    const [password, setPassword] = useState('')
+    const [password_confirmation, setPasswordConfirmation] = useState('')
+    const [showPassword, setShowPassword] = useState(false);
+    const [showPasswordConfirmation, setShowPasswordConfirmation] = useState(false);
+    const [registrationState, setRegistrationState] = useState({ error: false, message: '' })
+    const [, navigate] = useLocation()
+
+    const registrate = ({ email, password, password_confirmation }) => {
+      registrateService({ email, password, password_confirmation })
+      .then(res => {
+        setRegistrationState({ error: false, message: '' })
+        navigate('/login')
+      })
+      .catch(err => {
+        setRegistrationState({ error: true, message: err.message })
+      })
+    }
+
+    const submitHandler = (e) => {
+      e.preventDefault()
+      registrate({ email, password, password_confirmation })
+    }
+
+    const changeEmailHandler = (e) => {
+      setEmail(e.target.value)
+    }
+
+    const changePasswordHandler = (e) => {
+      setPassword(e.target.value)
+    }
+
+    const changePasswordConfirmationHandler = (e) => {
+      setPasswordConfirmation(e.target.value)
+    }
+
+    const changefullnameHandler = (e) => {
+      setFullnameHandler(e.target.value)
+    }
+
+    const handlePasswordToggle = () => {
+      if(showPassword){
+        setShowPassword(false)
+      } else {
+        setShowPassword(true)
+      }
+    }
```

Comment:
```suggestion
    const handlePasswordToggle = () =>  setShowPassword(!showPassword)
```

or if you want to go even more hacker:
```
const handlePasswordToggle = () =>  setShowPassword( prevState => !prevState)
```